homeassistant/components/unifiprotect/sensor.py

```diff
@@ -534,11 +533,11 @@ def _get_alarm_sound(obj: Sensor) -> str:
         key="smart_obj_licenseplate",
         name="License Plate Detected",
         icon="mdi:car",
-        device_class=DEVICE_CLASS_LICENSE_PLATE,
         ufp_value="is_smart_detected",
         ufp_required_field="can_detect_license_plate",
         ufp_event_obj="last_smart_detect_event",
         ufp_smart_type=SmartDetectObjectType.LICENSE_PLATE,
+        translation_key="license_plate",
```

Comment:
Can you move this above the custom, ufp keys?

Reply:
Sure!